.github/ISSUE_TEMPLATE.md

```diff
@@ -8,13 +8,12 @@
 
 [//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
```

Comment:
Was just traipsing through this issue to look at it--is it worth fixing this "Habitia" typo?

Reply:
ha! sure thing